### PR TITLE
adds emotion and material ui theme

### PR DESCRIPTION
**Pull request by mohit2530**
added material ui with emotion and cache. that was what we had previously before as well. we can change it later on if required.

### CODE REVIEW

**Comment by mohit2530** [comment]
@sujan-dahal the failure is i think for the `exit code 0` that we wrote so that it won't build !? I'm not quite sure. However, if you pull this down locally, `npm ci` and run it - it should work. I did verify that on my machine.

**Comment by mohit2530** [comment]
Also, i added the subscribe button with the email and now further commits down this line, I will edit them better.

**Comment by sujan-dahal** [comment]
@mohit2530 I am getting npm install errors on react 17 - 18 
Screenshot 2022-12-01 at 7 42 41 PM
Screenshot 2022-12-01 at 7 42 55 PM


We probably need a newer mui

File: components/Forms/AddFeatureForm.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import { Button, FormControl, FormControlLabel, FormHelperText, FormLabel, Input, InputLabel, Radio, RadioGroup, Snackbar, useMediaQuery } from "@mui/material";
+import { Box } from "@mui/system";
+import BodyHeaderContent from "../Home/BodyHeaderContent";
+import { useAddFeatureForm, useEmailForm } from "./CallToActionHook";
+
+const AddFeatureForm: React.FC = () => {
+    const mediumSizeOrHigher = useMediaQuery("(min-width:768px)");
```
Comment (sujan-dahal) [inline components/Forms/AddFeatureForm.tsx:7]: We should feature and expand on this min-width prop to get the dimensions right across all components on theme level.

**Review by sujan-dahal — APPROVED** [review]
I lied about the package install problem. Accidentally used npm instead of yarn.

**Pull request merged**